src/components/Theme/style.css

```diff
@@ -173,29 +104,23 @@
   }
 }
 
-.actions .btn-primary {
```

Comment:
This change is affecting the layout of the primary button in the confirm screen

Reply:
got it!

Reply:
fixed in commit https://github.com/onfido/onfido-sdk-ui/pull/552/commits/4eb4616d59ab86d1184c8603dc3a013d346eb0fe

and nice catch btw!